jerry-core/parser/js/js-parser-expr.c

```diff
@@ -369,7 +369,8 @@ parser_parse_class_literal (parser_context_t *context_p) /**< context */
   parser_emit_cbc (context_p, CBC_CREATE_OBJECT);
 
   bool super_called = false;
-  uint32_t status_flags = PARSER_IS_FUNCTION | PARSER_IS_CLOSURE | (context_p->status_flags & PARSER_CLASS_HAS_SUPER);
+  uint32_t status_flags = PARSER_IS_FUNCTION | PARSER_IS_CLOSURE;
+  status_flags |= (context_p->status_flags & (PARSER_CLASS_HAS_SUPER | PARSER_CLASS_IMPLICIT_SUPER));
```

Comment:
Now that the expression is broken out into a separate statement, the outer parentheses are unnecessary.
```c
status_flags |= context_p->status_flags & ...
```

Reply:
Exactly, the only problem with the single statement that it reaches the line length limit, but the outer parentheses can be removed.